src/views/UserSettings.vue

```diff
@@ -94,7 +94,34 @@ export default {
         this.languageTranslationCompletion = await localeManager.calculateTranslationCompletion(this.userSettingsForm.selectedLanguage.code)
       }
     },
-
+    'userSettingsForm.selectedCountry': function (newValue, oldValue) {
+      if (newValue !== oldValue) {
+        const selectedCountry = this.countryList.find(country => country.code === newValue)
+        // Update the currency list to the first currency of the selected country if it exists or USD otherwise
+        if (selectedCountry && selectedCountry.currency && selectedCountry.currency.length > 0) {
+          this.userSettingsForm.currency = selectedCountry.currency[0]
+        } else {
+          this.userSettingsForm.currency = 'USD'
```

Comment:
why is USD hardcoded here ? we have EUR set as default in the store. we could move everything to an env btw and have a `VITE_DEFAULT_CURRENCY` (like LOCALE & COUNTRY)

Reply:
I think I used that for testing and forgot to remove. The default should revert to VITE_DEFAULT_CURRENCY set in the store and the "else" statement is useless here

Reply:
It wasn't for testing. I changed to the store value